Podfile

```diff
@@ -23,7 +23,8 @@ workspace 'WooCommerce.xcworkspace'
 ## =====================================
 ##
 def aztec
-  pod 'WordPress-Editor-iOS', '~> 1.19.9'
+  # pod 'WordPress-Editor-iOS', '~> 1.19.9'
+  pod 'WordPress-Editor-iOS', :git => 'https://github.com/wordpress-mobile/AztecEditor-iOS.git', :commit => '499f9c294e76c2626cb1997f757cb429a07e5225'
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/woocommerce/woocommerce-ios/blob/61ecf78d986ddf9fe40e74b944e531acc3974a73/Podfile#L27"></span>Style/HashSyntax: Use the new Ruby 1.9 hash syntax.</td>
    </tr>
  </tbody>
</table>